src/screens/ContactScreen/index.js

```diff
@@ -106,17 +163,23 @@ const ContactScreen = ({navigation}) => {
   const handleCreateChannel = async () => {
     try {
       const mappingUserName = selectedUsers?.map((user) => user?.username).join(',');
+      const mappingUserId = selectedUsers?.map((user) => user?.user_id).join(',');
       let image = DEFAULT_PROFILE_PIC_PATH;
       if (selectedUsers.length === 1) {
         image = selectedUsers[0]?.profile_pic_path;
       }
       const dataSelected = {
         user: {
           name: mappingUserName,
-          image
+          image,
+          userId: mappingUserId
         }
       };
-      createSignChat(followed, dataSelected, 'CONTACT_SCREEN');
+      if (isAnon) {
+        createAnonymousChat(dataSelected);
+      } else {
+        createSignChat(followed, dataSelected, 'CONTACT_SCREEN');
+      }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `handleCreateChannel` function has been updated to handle anonymous chats. The `isAnon` check is used to decide whether to create an anonymous chat or a signed chat. This is a good practice as it allows the function to behave differently based on the context it is used in.
```diff
+ const mappingUserId = selectedUsers?.map((user) => user?.user_id).join(',');
+ let image = DEFAULT_PROFILE_PIC_PATH;
+ if (selectedUsers.length === 1) {
+   image = selectedUsers[0]?.profile_pic_path;
+ }
+ const dataSelected = {
+   user: {
+     name: mappingUserName,
+     image,
+     userId: mappingUserId
+   }
+ };
+ if (isAnon) {
+   createAnonymousChat(dataSelected);
+ } else {
+   createSignChat(followed, dataSelected, 'CONTACT_SCREEN');
+ }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->